Formula/stackql.rb

```diff
@@ -0,0 +1,29 @@
+class Stackql < Formula
+  desc "SQL interface for arbitrary resources with full CRUD support"
+  homepage "https://stackql.io/"
+  url "https://github.com/stackql/stackql.git",
+    tag:      "v0.3.265",
+    revision: "ce58a3b9cd00022da42cb8144922613e9491f2f3"
```

Comment:
```suggestion
  url "https://github.com/stackql/stackql.git",
      tag:      "v0.3.265",
      revision: "ce58a3b9cd00022da42cb8144922613e9491f2f3"
```
Align the arguments to `url`.

Reply:
@carlocab I've followed the suggested format changes.  As for tarball, `brew` locally throws errors on all permutations I have tried, so I am sticking with the git object pattern.